awx/ui_next/src/components/DetailList/NumberSinceDetail.jsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import { node, string } from 'prop-types';
+import styled from 'styled-components';
+import { formatDateString } from '../../util/dates';
+import _Detail from './Detail';
+
```

Comment:
```suggestion
import {t} from '@lingui/macro'
```

Reply:
```diff --git a/awx/ui_next/src/components/DetailList/NumberSinceDetail.jsx b/awx/ui_next/src/components/DetailList/NumberSinceDetail.jsx
index 617e7273b4..495e91b13f 100644
--- a/awx/ui_next/src/components/DetailList/NumberSinceDetail.jsx
+++ b/awx/ui_next/src/components/DetailList/NumberSinceDetail.jsx
@@ -1,5 +1,6 @@
 import React from 'react';
 import { node, string } from 'prop-types';
+import {t} from '@lingui/macro';
 import styled from 'styled-components';
 import { formatDateString } from '../../util/dates';
 import _Detail from './Detail';
@@ -15,7 +16,7 @@ function NumberSinceDetail({ label, number, date, dataCy = null }) {
     <Detail
       label={label}
       dataCy={dataCy}
-      value={`${number} since ${dateStr}`}
+      value={t`${number} since ${dateStr}`}
     />
   );
 }
```
![Screenshot from 2021-06-02 16-21-08](https://user-images.githubusercontent.com/5208768/120547164-bc94c100-c3be-11eb-9924-ad4f54f4c021.png)

@AlexSCorey Did I miss something somewhere?

Reply:
haha, probably not.  We updated a dependency to solve that issue last week so, depending on how you are building you likely need to update dependencies.